custom_components/circadian_lighting/switch.py

```diff
@@ -352,7 +353,8 @@ async def _adjust_lights(self, lights, transition):
             if not is_on(self.hass, light):
                 continue
 
-            service_data = {ATTR_ENTITY_ID: light, ATTR_TRANSITION: transition}
+            service_data = {ATTR_ENTITY_ID: light}
+            service_data[ATTR_TRANSITION] = transition
```

Comment:
Is there an advantage to breaking this into two lines? Have you run into issues where `service_data` was not set because of an issue with light or transition?

Reply:
Ah well spotted! I actually didn't intend to commit this, but I forgot to remove it after I tested it on my system. The problem is that IKEA light bulbs don't handle transition and color changes commands issued at the same time very well. Particularly, they don't change the color when this happens, so I just commented out the transition part on my system.